iterative_verifier.go

```diff
@@ -601,6 +602,11 @@ func compareHashes(source, target map[uint64][]byte) []uint64 {
 }
 
 func (v *IterativeVerifier) GetHashes(db *sql.DB, schema, table, pkColumn string, columns []schema.TableColumn, pks []uint64) (map[uint64][]byte, error) {
+	if tableCompression, ok := v.CompressionVerifier.tableCompressions[table]; ok {
```

Comment:
nitpick: I find it confusing that there are two things that are both called `tableCompression` but one is a `map[string]map[string]string` while the other is a `map[string]string`. Can we use different terminology? Maybe one is called `tableCompressions` and the other (the smaller one) is called `columnCompressions`?